template/apps/web/src/config/environment/development-docker.json

```diff
@@ -1,5 +1,16 @@
 {
   "apiUrl": "http://localhost:3001",
   "wsUrl": "http://localhost:3001",
-  "webUrl": "http://localhost:3002"
+  "webUrl": "http://localhost:3002",
+  "stripePublicKey": "pk_test_51Lp7VnIqrmVCYktKsZAzKncNrvpo4Zsf4hVFKF7jyML3kUnuLZfDSIoNCUWWC4IVNLY6ywmCEYRFT5yUgpR9wSWp000JmvSmOU",
```

Comment:
I think we can configure `stripePublicKey` and `subscriptions` only inside `environment/index.ts` instead dev, staging, prod, ...